### DISCUSSION TITLE
CPU Usage and Temp

### DISCUSSION THREAD

**Discussion by Cpd5899** [hyperion-project/hyperion.ng#1680] [Ideas]
I had been using HyperHDR prior to trying out Hyperion. One thing I truly liked about HyperHDR was the ability to see the CPU usage and temp.

**Comment by Paulchen-Panther** [comment]
Hyperion is an open source project. You are free to participate and/or create a feature/pull request at any time.
As the development team is unfortunately completely understaffed, we will not be able to fulfil your wishes in the near future.

With kind regards Paulchen
